### DISCUSSION TITLE
运行时内存超过阈值，怎么解决？

### DISCUSSION THREAD

**Discussion by justsoso0601** [secretflow/secretflow#799] [Q&A]
<img width="882" alt="image" src="https://github.com/secretflow/secretflow/assets/92526946/4e247add-5da5-4d6d-9975-166450066995">

**Comment by anakinxc** [comment]
@zhouaihui @ian-huu can you take a look?

Thanks

**Comment by zhouaihui** [comment]
My suggestion is giving it a larger memory.
